### PR TITLE
update to polymer 0.9.0-rc.1

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata @notwaldorf PTAL

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "homepage": "https://github.com/PolymerElements/iron-validatable-behavior",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#v0.8.0-rc.7",
+    "polymer": "Polymer/polymer#v0.9.0-rc.1",
     "iron-meta": "PolymerElements/iron-meta#^0.8.0"
```
Comment (notwaldorf) [inline bower.json]: We've been bumping all the elements to 0.9.0 :(

File: test/simple-validator.html [hunk 2]
```diff
@@ -1,26 +0,0 @@
-<!--
-@license
```
Comment (notwaldorf) [inline test/simple-validator.html:2]: no moar test?

Comment (morethanreal) [inline test/simple-validator.html:2]: That's an unused file that I checked in by mistake in the last commit

**Comment by morethanreal** [comment]
Updated element deps.

**Comment by morethanreal** [comment]
ping?

**Comment by notwaldorf** [comment]
oh, sorry! lgtm! :balloon:

**Pull request merged**